www/source/blog/2018-05-11-changes-in-the-0.56.0-supervisor.html.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Changes in the Forthcoming 0.56.0 Supervisor: What You Need To Know
+date: 2018-05-11
+author: Christopher Maier
+tags: supervisor
+classes: body-article
+---
+
+The 0.56.0 release of Habitat brings with it one of the largest changes to the Supervisor that we've ever made. We'd like to share with you what these changes are, why we needed to make them, and what steps you as a Supervisor user need to take to deal with these changes. They're big changes, but they will unlock a lot of exciting possibilities for us in the future.
+
+Before the 0.56.0 release, the communication protocol between the `hab` commandline tool and a running Supervisor essentially consisted of modifying files on disk. While this is a very simple interaction model to implement and think about, it leaves much to be desired. It is difficult to achieve synchronous interactions between process. It is difficult to validate interactions and abort them in the face of errors. It has introduced several difficult-to-reason-about edge-case bugs. It is inherently restricte to "same machine" interaction patterns. For these reasons and more, we needed to devise a new communication protocol.
```

Comment:
two minor things (missing 's' and 'd'):
 • s/interactions between _process_./interactions between processes./
 • s/It is inherently _restricte_ to/It is inherently restricted to/

Reply:
@jeremymv2 Thanks for catching those! I'll get those fixed up.